cli/cmd/install/install.go

```diff
@@ -198,11 +221,52 @@ timescaledb-single:
 		helmValues = appendPrometheusHAValues(helmValues)
 	}
 
+	if c.enableOtel {
+		helmValues = enableOtelInValues(helmValues)
+	} else {
+		e, err := helmClient.ExportValuesFieldFromChart(c.Ref, c.ConfigFile, []string{"opentelemetryOperator", "enabled"})
+		if err != nil {
+			return err
+		}
+		var ok bool
+		c.enableOtel, ok = e.(bool)
+		if !ok {
+			return fmt.Errorf("enable otel was not a bool")
+		}
+	}
+
+	if c.enableOtel {
+		// opentelemetry operator needs cert-manager as a dependency as adding cert-manager isn't good practice and
+		// not recommended by the cert-manager maintainers. We are explicitly creating cert-manager with kubectl
+		// for more details on this refer: https://github.com/jetstack/cert-manager/issues/3616
+		apiClient := k8s.NewAPIClient()
+		_, err = apiClient.GetCRD("certificates.cert-manager.io")
+		if err != nil {
+			if errors.IsNotFound(err) {
+				fmt.Println("Couldn't find the cert-manager, Installing the cert-manager...")
+				utils.ConfirmAction()
+				err = k8s.CreateK8sManifests([]string{"https://github.com/jetstack/cert-manager/releases/download/v1.5.2/cert-manager.yaml"})
```

Comment:
Let's abstract that link as a `const` and put it at the top, so we can easily swap it later on.